book/http_cache.rst

```diff
@@ -856,7 +856,7 @@ If one content corresponds to one URL, the ``PURGE`` model works well.
 You send a request to the cache proxy with the HTTP method ``PURGE`` (using
 the word "PURGE" is a convention, technically this can be any string) instead
 of ``GET`` and make the cache proxy detect this and remove the data from the
-cache instead of going to Symfony to get a response.
+cache instead of going to the application to get a response.
```

Comment:
The topic mostly uses *application* and does tie itself to Symfony, as this part is more about general working of cache I think it's a good idea to also just use application